main.go

```diff
@@ -464,10 +484,9 @@ func CreateQuickstartTests(quickstartName string) bool {
 						T.CreatePullRequestAndGetPreviewEnvironment(200)
 					}
 
-					if T.DeleteApplications() {
+					if T.DeleteApps() {
 						By("deletes the application")
-						fullApplicationName := T.GetGitOrganisation() + "/" + T.ApplicationName
-						args = []string{"delete", "application", "-b", fullApplicationName}
+						args = []string{"delete", "application", "-b", T.AppName}
```

Comment:
I think with recent changes merged the other day, this needs to be `T.ApplicationName`